experimental/sdg-tracker/src/components/countries/CountriesContent.tsx

```diff
@@ -332,49 +332,50 @@ interface Goals {
 const ChartCategoryContent: React.FC<{
   chartConfigCategory: ChartConfigCategory;
   placeDcid: string;
-  varToTopic: VarToTopicMapping;
-}> = ({ chartConfigCategory, placeDcid, varToTopic }) => {
+  varToTopics: VarToTopicMapping;
+}> = ({ chartConfigCategory, placeDcid, varToTopics }) => {
   // stores hierarchy of Goals -> Target -> Indicator -> Tiles
   const allGoals: Goals = {};
 
   // iterate over tiles nested in chartConfigCategory
   chartConfigCategory.blocks.forEach((block) => {
     block.columns.forEach((column) => {
       column.tiles.forEach((tile) => {
-        // Find which goal, target, and indicator this tile belongs to
-        const topicDcid = !_.isEmpty(tile.statVarKey)
-          ? varToTopic[tile.statVarKey[0]].dcid
-          : "";
-        const indicatorMatches = topicDcid.match(
-          /dc\/topic\/sdg_(\d\d?\.\w\w?\.\w\w?)/
-        );
-        const targetMatches = topicDcid.match(/dc\/topic\/sdg_(\d\d?\.\w\w?)/);
-        const goalMatches = topicDcid.match(/dc\/topic\/sdg_(\d\d?)/);
-        const indicator =
-          indicatorMatches && indicatorMatches.length > 1
-            ? indicatorMatches[1]
-            : "none";
-        const target =
-          targetMatches && targetMatches.length > 1 ? targetMatches[1] : "none";
-        const goal =
-          goalMatches && goalMatches.length > 1 ? goalMatches[1] : "none";
-
-        // put tile in appropriate spot in allGoals
-        if (goal in allGoals) {
-          if (target in allGoals[goal]) {
-            if (indicator in allGoals[goal][target]) {
-              allGoals[goal][target][indicator].push(tile);
+        if (!_.isEmpty(tile.statVarKey) && !_.isEmpty(varToTopics[tile.statVarKey[0]])) {
+          for (const topic of varToTopics[tile.statVarKey[0]]) {
```

Comment:
For my understanding, will a page show the same chart now under different topics?  Or will something downstream only show a specific indicator?

Reply:
Ok.

If the LHS is showing Goal 1 though, it doesn't make a lot of sense to show anything related to 2.2 on RHS right?

Reply:
(Not for today)

Reply:
Agreed, let's get this change in and I'll add more filtering in a followup PR.